tests/manage/z_cluster/cluster_expansion/test_osd_topology_spread.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from _collections import defaultdict
+
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.node import get_ocs_nodes, get_osds_per_node, get_node_name
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    skipif_ocs_version,
+    skipif_ocp_version,
+    tier1,
+)
+
+
+@tier1
+@skipif_ocs_version("<4.7")
+@skipif_ocp_version("<4.6")
+@pytest.mark.last
+class TestOsdTopologySpread(ManageTest):
```

Comment:
I think this test should be incorporated as a validation step post-deployment and post other test cases that involve OSD operation, such as add and replace.

@kesavanvt @nehaberry @hnallurv what do you think?

Reply:
If i am not wrong, only the add_capacity test of tier 1 adds alters the OSD, this Tier 1 test (topology spread) always runs at the last , thus validates the add_capacity test. If this needs to be added as a validation step for the post deployment and add_capacity test, please help in providing me the possible changes required. @ebenahar @nehaberry @hnallurv

Reply:
@ramkiperiy @keesturam

Reply:
I assume that domin was originally meant to be domain, but remains mis-spelled because we don't want to have to rename the function all the other code that calls it.